README.md

```diff
@@ -129,8 +129,48 @@ New tenants will come with some basic initial data already installed, but if you
 3. use Ctrl + D or `exit()` to close the Python shell.
 
 
-## Contributing
+### Enabling Google Sign In (Optional)
 
-See [CONTRIBUTING.md](https://github.com/bytedeck/bytedeck/blob/develop/CONTRIBUTING.md) if you plan to contribute code to this project.  It contains critical information for your pull request to be accepted and will save you a lot of time!
 
+Here are the steps, assuming that you now have a functional tenant:
+
+1. Obtain Google credentials: https://developers.google.com/workspace/guides/create-credentials#oauth-client-id
+2. Make sure that in the Authorized URIs, add `http://hackerspace.localhost.net:8000/accounts/google/login/callback/`. We will explain why we are using `localhost.net` later but for now, just add this.
+3. Go to Social Applications: http://localhost:8000/admin/socialaccount/socialapp/
+4. Click Add Social Application
+5. Fill in `Client Id` and `Secret Key`. And then add the `Available Sites` to `Chosen Sites`
+6. Click Save
+7. Go to your Site Configuration: http://hackerspace.localhost:8000/config/ and click `Enable sign-in via Google`
```

Comment:
This step needs to be updated

Reply:
> > I'm slowly working my way through the code and testing stuff. One bug I think I came across:
> 
> I'm getting this error even after I enable Google Sign in on the domain. Maybe I screwed something up or missed a step?

Try upgrading django-allauth to the one specified in the requirements file. This happened to me when I was using the old version

Reply:
Is there a way we can automatically catch outdated dependancies?  I don't know what kind of hook or test or..something that would be.  Something that automatically re-checks or attempts to reinstall requirements.txt.  Any ideas?